src/jobs/jobs.controller.ts

```diff
@@ -0,0 +1,73 @@
+import { Request, Response } from 'express';
+import * as JobsModel from './jobs.model';
+import axios from 'axios';
+import e from 'cors';
+import { deprecate } from 'util';
+
+export const getAllJobs = async (req: Request, res: Response) => {
+  try {
+    const jobs = await JobsModel.getJobs();
+    res.status(200).json(jobs);
+  } catch (err: any) {
+    res.status(500).json({ message: err.message });
+    console.error(err.message);
+  }
+};
+
+export const getSingleJob = async (req: Request, res: Response) => {
+  try {
+    const id = parseInt(req.params.id);
+    if (!id) {
+      res.status(400).json({ message: 'Invalid job ID.' });
+    }
+    const singleJob = await JobsModel.getJobById(id);
+    if (!singleJob) {
+      res.status(400).json({ message: 'Job not found.' });
+    } else {
+      res.status(200).json(singleJob);
+    }
+  } catch (err: any) {
+    console.error(err.message);
+    res.status(500).json({ message: err.message });
+  }
+};
+
+export const addSingleJob = async (req: Request, res: Response) => {
+  try {
+    const data = req.body;
+    const addDepartment = await JobsModel.addNewJob(data);
+    res.status(200).json(addDepartment);
+  } catch (err: any) {
+    console.error(err.message);
+    res.status(500).json({ message: err.message });
+  }
+};
+
+export const deleteSingleJob = async (req: Request, res: Response) => {
+  try {
+    const id = parseInt(req.params.id);
+    const deleteJob = await JobsModel.deleteJob(id);
+    if (!deleteJob) {
+      res.status(400).json({ message: 'That job ID does not exist' });
```

Comment:
Please change to " `Job ${id} not found in database.` "

Reply:
Done.